.github/workflows/update-css.yml

```
@@ -0,0 +1,89 @@
+name: Update CSS
+
+on:
+  release:
+    types: [published]
+    repositories:
```

actionlint (1.7.4)

unexpected key "repositories" for "release" section. expected one of "branches", "branches-ignore", "paths", "paths-ignore", "tags", "tags-ignore", "types", "workflows"

(syntax-check)